### DISCUSSION TITLE
Constructing a new collapse point of thorium M106.0

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#22] [Q&A]
When the video is played, the search engine is switched. When the search page is in the queue of page tab 1, when it is switched from sogou search in China to Baidu, the flash back or crash will be triggered with high probability.

**Comment by gz83** [comment]
> When the video is played, the search engine is switched. When the search page is in the queue of page tab 1, when it is switched from sogou search in China to Baidu, the flash back or crash will be triggered with high probability.

@whxuezhe Is it convenient to record an operation video? Let's see if we can reproduce the problem you're having.

**Comment by Alex313031** [comment]
@gz83 Might be an extension he has, this isn't normal behaviour

**Reply by whxuezhe** [reply]
@gz83 There is a reason for this, but I only installed extensions that are mainly used for ad filtering；And now I find that when page 1 is Bing search home page, 2 and 3 are Sogou search and 3 respectively. When playing video, switch 2 to another refutation video website. When the address bar URL switching jump is applicable, there is a certain chance that it will flash back and crash.

**Reply by gz83** [reply]
> @gz83 There is a reason for this, but I only installed extensions that are mainly used for ad filtering；And now I find that when page 1 is Bing search home page, 2 and 3 are Sogou search and 3 respectively. When playing video, switch 2 to another refutation video website. When the address bar URL switching jump is applicable, there is a certain chance that it will flash back and crash.

@whxuezhe Have you tried testing with all extensions turned off?
